layers/enums.go

```diff
@@ -93,7 +93,7 @@ const (
 type LinkType uint8
 
 const (
-	// According to pcap-linktype(7) and http://www.tcpdump.org/linktypes.html
+	// According to pcap-linktype(7) and http://www.tcpdump.org/linktypes.html with fixes from pcap/bpf.h
```

Comment:
Sadly, it appears that raw can be 12 or 14 (https://opensource.apple.com/source/libpcap/libpcap-18/libpcap/pcap-bpf.h) or 101 https://github.com/the-tcpdump-group/libpcap/blob/1a6b088a88886eac782008f37a7219a32b86da45/pcap-common.c#L159.  Note that 101 is canonical, since it's how libpcap says "this could be one of a few values".  I wonder if we should do:

```go
  LinkTypeRaw 101
  LinkTypeRawBSD 14
  LinkTypeRawLinux 12
```

Thoughts?  Without 101 pointing at raw, pcap files from raw interfaces will no longer work.